### DISCUSSION TITLE
:PrtChatNew always connects to Haiku after I've selected Opus

### DISCUSSION THREAD

**Discussion by benwoodward** [frankroeder/parrot.nvim#5] [Q&A]
First of all, thanks for making this, it's fantastic.

I invoked `:PrtAgent` and hit enter on the opus agent, however, when I invoke `:PrtChatNew` it always connects to Haiku ... am I doing something wrong?

**Comment by benwoodward** [comment]
The problem was that I was typing `:PrtAgent<CR>` when I was supposed to type `:PrtAgent <tab>` then select from the list.

![image](https://github.com/frankroeder/parrot.nvim/assets/1472981/4055ccfd-b1fb-49ce-924a-4c58c66acba4)

**Comment by frankroeder** [comment]
Thank you for your feedback. I hope that the [commit](https://github.com/frankroeder/parrot.nvim/commit/6b9c23ed1dcd07b0fdb262d0e50d8952529f476e) makes things clearer.
